README.md

```diff
@@ -0,0 +1,68 @@
+cerner_splunk_ingredient Cookbook
+=================================
+Resource cookbook which provides custom resources for installing Splunk.
+
+These resources can:
+- Install Splunk via downloadable package or archive
+
+Requirements
+------------
+Supports Linux and Windows based systems with package support for Debian, Redhat,
+and Windows.
+
+---
+
+Resources
+---------
+
+### splunk_install
+Installs Splunk. Easy.
```

Comment:
I wanted to go the route of nouns and verbs, but opted for splunk_install as it was probably going to be a huge resource to cram configure in with it. Chances are, though, that the resource-side of things isn't going to be terribly complicated. I fully intend to rename the resource to just 'splunk' if I add configuration and it's clean enough.